contracts/test/TestRpcCompatibility.sol

```diff
@@ -21,4 +30,15 @@ contract TestRpcCompatibility {
         subscribeValue = newValue;
         emit ValueUpdatedForSubscribe(newValue);
     }
+
+    function emitCallContractWithToken(
```

Comment:
Why add another event? We already have a method that emits an event, and we don't care about the event content for this test